internal/state/module_test.go

```diff
@@ -349,10 +349,10 @@ provider "blah" {
 		t.Fatal(err)
 	}
 
-	expectedParsedFiles := map[string]*hcl.File{
+	expectedParsedConfigFiles := map[string]*hcl.File{
```

Comment:
Nitpick: as mentioned I'd suggest `expectedParsedModuleFiles`, "config" could technically be even tfvars config